moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -275,11 +292,11 @@ const MealReservation = (): JSX.Element => {
                                         key={uuid()}
                                         active={active === activeTabIndex}
                                         onClick={() => {
-                                            setDeyOfWeek(date.day())
-                                            setActive(activeTabIndex)
                                             if (cartItems.length) {
                                                 handleTabClickWithCartItems()
                                             } else {
+                                                setDeyOfWeek(date.day())
+                                                setActive(activeTabIndex)
                                                 setActiveDate(day.date)
                                             }
```

Comment:
lets avoid `if - else` if we can, and return

`if (...) return ...`
`if (...) return ...`